website/project/model.py

```diff
@@ -3980,3 +4028,9 @@ def reject(self, user):
     def add_status_log(self, user, action):
         log = DraftRegistrationLog(action=action, user=user, draft=self)
         log.save()
+
+    def validate_metadata(self, *args, **kwargs):
+        """
+        Validates draft's metadata
+        """
+        return self.registration_schema.validate_metadata(*args, **kwargs)
```

Comment:
When I am creating or updating a draft, I am running validate_metadata before I run update_metadata.  In the instance where I am creating a draft, there won't be any metadata yet. Where I am updating the draft, the metadata will be outdated.  This only works when I am validating the metadata when creating the actual registration and the draft's metadata is not changing.